x/gov/types/hooks.go

```diff
@@ -0,0 +1,40 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
```

Comment:
```suggestion
var _ GovHooks = new MultiGovHooks{}
```

Good to make sure that the interface is satisfied